src/views/auth/Register.vue

```diff
@@ -66,8 +68,14 @@ export default {
     };
   },
   methods: {
+    ...mapActions(['register']),
     registerUser: function () {
-
+      this.$store.dispatch('register', {
+        name: 'gwenf',
+        email: 'gwenfaraday@protonmail.com',
+        password: 'o6qSOp5eO7U8'
+      })
+      // this.$router.push('/github-auth');
```

Comment:
this seems suspect

Reply:
lol. Don't worry, it's not my password. I just tried it temporarily for something. I need to remove it.